recipes/VideoBots.py

```diff
@@ -316,13 +316,18 @@ def related_workflows(self):
     def get_run_title(cls, sr: SavedRun, pr: PublishedRun | None) -> str:
         import langcodes
 
+        title = super().get_run_title(sr, pr)
+        root_title = cls.get_recipe_title()
+        if title != root_title:
```

Comment:
this comparision seems like not the thing you wanted to check here, its really checking if `pr and pr.title` implicitly